GMX-Bots/Detect-many-closing/src/agent.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Finding, HandleTransaction, ethers } from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+import { createAddress, MockEthersProvider, TestTransactionEvent } from "forta-agent-tools/lib/tests";
+import { provideHandleTransaction } from "./agent";
+import utils from "./utils";
+
+const WRONG_EVENT_ABI: string[] = ["event Transfer(address indexed from,address indexed to,uint256 value)"];
+
+const WRONG_EVENTS_IFACE: ethers.utils.Interface = new ethers.utils.Interface(WRONG_EVENT_ABI);
+
+const TEST_GMX_VAULT: string = createAddress("0xcdcd");
+
+const TEST_NETWORK_DATA = {
+  0: {
+    address: TEST_GMX_VAULT,
+    positionsNumber: 50,
+    blockNumber: 100,
+  },
+};
+
+const CASES = [
```

Comment:
Rename this to `EXPECTED_FINDINGS`. 
But I don't see the need to have those separated in here, would be better to have each expected finding inside its test case imo.